linux_os/guide/system/auditing/auditd_configure_rules/file_group_ownership_var_log_audit/oval/shared.xml

```diff
@@ -0,0 +1,46 @@
+<def-group>
+  <definition class="compliance" id="file_group_ownership_var_log_audit" version="1">
+    {{{ oval_metadata("Checks that all /var/log/audit files are group owned by the root user.") }}}
+    <criteria operator="OR">
+      <criteria operator="AND" comment="files are root group owned">
+        <criterion test_ref="test_group_ownership_var_log_audit_files" />
+      </criteria>
```

Comment:
The criteria element is superfluous because it contains just a signle criterion.

Reply:
Ok, I'll remove it